ttnn/ttnn/decorators.py

```diff
@@ -694,6 +694,10 @@ def __call__(self, *function_args, **function_kwargs):
             OPERATION_CALL_STACK.pop()
         return output
 
+    def __eq__(self, other: callable):
```

Comment:
Is this needed for the changes?

Reply:
I second this question

Reply:
Upon further testing, I don't think this is needed anymore. I'll remove it.